aiorussound/rio.py

```diff
@@ -232,6 +237,95 @@ async def init_sources(self) -> None:
             except CommandError:
                 break
 
+    async def enumerate_zone_favorites(self, zone: Zone) -> list[Favorite]:
```

Comment:
To maintain the object paradigm, let's move this method to the zone class and rename it `enumerate_favorites`, and remove the zone parameter. It will just return the favorites for that zone.